source/includes/pipeline_config/_002-environment-variable.md.erb

```diff
@@ -0,0 +1,27 @@
+<%
+json ='{
+  "environment_variables": [
+    {
+      "name": "COMPLEX",
+      "value": "plain",
+      "secure": false
+    },
+    {
+      "name": "MULTIPLE_LINES",
+      "encrypted_value": "1f3rrs9uhn63hd",
+      "secure": true
+    }
```

Comment:
please use something more meaningful here — `USERNAME` and `PASSWORD` or `AWS_SECRECT_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are probably good examples.